static/js/global.js

```diff
@@ -26,7 +26,8 @@ function updateTranslation(localex) {
 $(window).on('load', function() {
   var localeOptions = {
     English: 'en',
-    Español: 'es',
+    Español: 'es'
```

Comment:
You'll need a comma here, this will crash as-is.

Reply:
`vi` started to behave really strange on my phone -_-

Reply:
You submitted a PR from your `master` branch :P

Reply:
I know, @jayvdb… I don’t need to create branches, because I’m not going to change anything more in this repo…